docs/installation/mac.md

```diff
@@ -61,11 +62,10 @@ redis-cli ping
 
 ### Elasticsearch
 
-Forem requires a version of Elasticsearch between 7.1 and 7.5. Version 7.6 is
+Forem requires a version of Elasticsearch between 7.1 and 7.5 to be running. Version 7.6 is
 not supported. We recommend version 7.5.2.
```

Comment:
I think we should update this. In production we run 7.5.2 (though it technically runs with 7.9 as well but it's not officially supported), we don't really know if it still works on 7.1 to 7.4.

Maybe we should mention 7.5 or higher, like we do for PostreSQL?